conf.sh

```diff
@@ -0,0 +1,18 @@
+PYPY_REGIONS=(
+  #ap-northeast-1
+  #ap-northeast-2
+  #ap-south-1
+  #ap-southeast-1
+  #ap-southeast-2
+  #ca-central-1
+  eu-central-1
```

Comment:
Do we need to re-enable other regions for publish?

Reply:
Not unless we intend to release the layer in a public manner world wide
(which we don't atm)